### PR TITLE
test: fix tests by adding missing test scenarios for regional dr charts

### PR DESCRIPTION
**Pull request by TomerFi**
Signed-off-by: Tomer Figenblat <tfigenbl@redhat.com>

### CODE REVIEW

File: tests/common-clustergroup-industrial-edge-hub.expected.yaml [hunk 1]
```diff
@@ -368,6 +368,95 @@ data:
...
+                  project: ValidatedPatterns
+            publish: External
+            pullSecret: ""
+            sshKey: ""
+          name: ocp-primary
+          version: 4.14.12
+        secondary:
+          install_config:
+            apiVersion: v1
+            baseDomain: fsi-partner.rhecoeng.com
```
Comment (r2dedios) [inline tests/common-clustergroup-industrial-edge-hub.expected.yaml]: Replace baseDomain by "example-domain.com"

**Review by r2dedios — CHANGES_REQUESTED** [review]
Everything looks good. I think we should re-generate the tests files removing the baseDomain

File: tests/common-clustergroup-industrial-edge-hub.expected.yaml (hunk 1 above)
Comment (TomerFi) [reply tests/common-clustergroup-industrial-edge-hub.expected.yaml]: Good catch!

Comment (TomerFi) [reply tests/common-clustergroup-industrial-edge-hub.expected.yaml]: Fixed in https://github.com/RHEcosystemAppEng/regional-resiliency-pattern/pull/6/commits/cdf4acff07188bf8d49e80a4bab178ef3193ca6e.

**Review by r2dedios — APPROVED** [review]
Looks great ;)

**Pull request merged**